_posts/Use-your-artistic-skills-to-help-open-source

```diff
@@ -0,0 +1,3 @@
+Repost from OpenSource.com for an article from [Jeff Macharyas](https://opensource.com/users/jeffmacharyas) about Open Source Software projects and how Open Source Design connects projects that need design help with the designers who can help them.
```

Comment:
This needs the proper jekyll metadata.

Reply:
Do you mean the part that looks like this:

---
layout: post
title: "Welcome to Open Source Design"
date: 2015-04-25
categories: design open source launching
author: bnvk
redirect_from:
  - /launching
permalink: /2015/04/25/welcome-to-open-source-design
---

Reply:
Yeah